### PR TITLE
Update JSON Schema & add a test to ensure privacyAgreementAccepted=true

### PR DESCRIPTION
**Pull request by markolson**

### CODE REVIEW

File: src/common/veteran.js [hunk 1]
```diff
@@ -138,7 +138,8 @@ const blankVeteran = {
   vietnamService: false,
   exposedToRadiation: false,
   radiumTreatments: false,
-  campLejeune: false
+  campLejeune: false,
+  privacyAgreementAccepted: true
```
Comment (akainic) [inline src/common/veteran.js]: Should this be set to `false` first, since this is for an initial blank application?

**Review by akainic — COMMENTED** [review]
Looks good except for one question!

File: src/common/veteran.js (hunk 1 above)
Comment (markolson) [inline src/common/veteran.js]: Ok

**Review by akainic — APPROVED** [review]
LGTM!

**Pull request merged**